src/components/Form.js

```diff
@@ -34,28 +34,30 @@ const Input = () => {
 
   return (
     <form className="input-area" onSubmit={BookSubmit}>
-      <h2>Add books</h2>
-      <div className="title">
-        <input
-          type="text"
-          id="text"
-          placeholder="Book Title"
-          // value={booksState.title}
-          onChange={addNewBook}
-        />
+      <h2 className="add-book">Add books</h2>
+      <div className="imput-cont">
+        <div className="title">
+          <input
+            type="text"
+            id="text"
+            placeholder="Book Title"
+            // value={booksState.title}
+            onChange={addNewBook}
+          />
+        </div>
+        <div className="author">
+          <input
+            type="text"
+            id="text-author"
+            placeholder="Book Author"
+            // value={booksState.author}
```

Comment:
- [ ] For codes that you're not using in your project, it's best to delete or remove them so your codebase looks clean and professional. It's a good practice to not have commented codes in our codebase and that makes your project clean and professional. Kindly remove this 👍🏽